packages/block-library/src/site-logo/edit.js

```diff
@@ -166,7 +166,8 @@ const SiteLogo = ( {
 	const currentWidth = width || defaultWidth;
 	const ratio = naturalWidth / naturalHeight;
 	const currentHeight = currentWidth / ratio;
-	const minWidth = naturalWidth < naturalHeight ? MIN_SIZE : MIN_SIZE * ratio;
+	const minWidth =
+		naturalWidth < naturalHeight ? MIN_SIZE : Math.ceil( MIN_SIZE * ratio );
 	const minHeight =
 		naturalHeight < naturalWidth ? MIN_SIZE : MIN_SIZE / ratio;
```

Comment:
Should we round this value as well?

Reply:
Thanks @Mamaduka! I was thinking same. Did some more testing and do see that the `minHeight` isn't a integer on portrait images.

I update the PR.